docs/reference/api/docker_remote_api_v1.14.md

```diff
@@ -126,7 +126,11 @@ Create a container
              "Tty":false,
              "OpenStdin":false,
              "StdinOnce":false,
-             "Env":null,
+             "Env": [
+                     "FOO=bar",
+                     "FOO=baz",
```

Comment:
I'm not sure we should encourage people to add info that will be ignored - unless you want to specifically talk about this edge case some place in the docs. But as a "normal" example, I wouldn't add it.